privilege-management.md

```diff
@@ -103,9 +157,9 @@ In this example, because of the `%` in `te%`, all the databases starting with `t
 
 ### Revoke privileges
 
-The `REVOKE` statement enables system administrators to revoke privileges from the user accounts.
+The [`REVOKE`](/sql-statements/sql-statement-revoke-privileges.md) statement enables system administrators to revoke privileges from the user accounts.
 
-The `REVOKE` statement corresponds with the `REVOKE` statement:
+The [`REVOKE`](/sql-statements/sql-statement-revoke-privileges.md) statement corresponds with the [`REVOKE`](/sql-statements/sql-statement-revoke-privileges.md) statement:
```

Comment:
```suggestion
The `REVOKE` statement corresponds with the `REVOKE` statement:
```

Including too many links can distract from the main content or steal people's focus. It is recommended to do not repeat the same link more than once in the same article or section.